thin_client/test/test_networking.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import thin_client.networking as networking
+import socket
+
+class test_networking(unittest.TestCase):
+    # Check if packAndSend increments the sequence number correctly       
+    def test_packAndSend_sequence(self):
```

Comment:
separate each of these tests into separate unit test method. provided each request is independent.